docs/content/kv_commands.md

```diff
@@ -62,3 +60,89 @@ $ wrangler kv list
 | new kv namespace | f7b02e7fc70443149ac906dd81ec1791 |
 +------------------+----------------------------------+
 ```
+
+## `kv:key`
+
+### `put`
+
+Writes a single key/value pair to the given namespace.
+
+#### Usage
+
+```sh
+$ wrangler kv:key put "key" "value" --ttl=10000 --namespace=f7b02e7fc70443149ac906dd81ec1791
+```
+
+### `get`
+
+Reads a single value by key from the given namespace.
+
+#### Usage
+
+```sh
+$ wrangler kv:key get "key" --namespace=f7b02e7fc70443149ac906dd81ec1791
```

Comment:
We should update these instructions to replace `--namespace=` with just a positional argument for namespace (at least until I #502 is unblocked and done)

Reply:
done.